.github/workflows/continuous-integration.yml

```diff
@@ -23,12 +23,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout open-amp
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
```

Comment:
"path: open-amp" HERE ???
with adjustments in script?

I think libmetal and openamp-system-reference are sub directories of open-amp.  That works but is a little strange.  Is this easy to fix as described above??  If not we will deal with it later.

Reply:
I take this comment back.  Every build would need to be updated not just the linux build.
We should do this in another PR